### ISSUE TITLE
Suppress R package output ("noise")

### ISSUE THREAD

**Issue by lbeltrame** [lpantano/bcbio.coverage#4]
The code, if possible, should be adjusted to reduce stray output as much as possible from R packages, which are quite chatty by default.

**Comment by lpantano** [comment]
Hi @lbeltrame,

do you mean some message some functions print? 

thanks for all your feedback as always.

**Comment by lbeltrame** [comment]
> do you mean some message some functions print?

Yes, I mean warning statements and similar things - R packages are very chatty 
unfortunately.
